tools/autograd/gen_python_functions.py

```diff
@@ -212,6 +212,7 @@ def create_python_bindings(python_functions, has_self, is_module=False):
         'int64_t': 'toInt64',
         'bool': 'toBool',
         'double': 'toDouble',
+        'std::string' : 'string',
```

Comment:
nit: I think that once https://github.com/pytorch/pytorch/pull/6283 is merged, you will be able to use [`String`](https://github.com/pytorch/pytorch/pull/6283/files#diff-54816789e967b71e1be8048e53f8ccbdR29)

Reply:
Thak you for the pointer! I will rebase on that eventually.